src/def.d.ts

```diff
@@ -464,6 +464,9 @@ interface VendettaObject {
     DISCORD_SERVER_ID: string;
     PLUGINS_CHANNEL_ID: string;
     THEMES_CHANNEL_ID: string;
+    DISCORD_REVENGE_SERVER_ID: string;
+    PLUGINS_REVENGE_CHANNEL_ID: string;
+    THEMES_REVENGE_CHANNEL_ID: string;
```

Comment:
Don't think we need to put type definitions for the new constants here yet, maybe after #29 is completed we can put our own APIs definitions.

Reply:
Got it, then in that case I'll wait for 29 to be completed and then discuss about it. To be completely honest, I once managed to get it work without defining the constants in def.d.ts, but I don't know what black magic there was.

Reply:
Oh no, #29 isn't blocking this PR. The type definitions provided in the d.ts file is just for generating the types package IIRC. It should work without the new type additions.